teleport/datadog_checks/teleport/__about__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+__version__ = '1.0.0'
```

Comment:
```suggestion
__version__ = '0.0.1'
```
this is a dummy value, we'll bump it to 1.0.0 when we release the integration.